docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TreeFeaturization/FastForestBinaryFeaturizationWithOptions.cs

```diff
@@ -0,0 +1,139 @@
+                Console.WriteLine($"  Leave IDs' 0-1 representation: [{String.Join(",", transformedDataPoint.Leaves)}].");
+                Console.WriteLine($"  Paths IDs' 0-1 representation: [{String.Join(",", transformedDataPoint.Paths)}].");
+            }
+
+            // Expected output:
+            //  The original feature vector [0.8173254,0.7680227,0.5581612] is transformed to three different tree-based feature vectors:
+            //    Trees' output values: [0.1111111,0.8823529].
+            //    Leave IDs' 0-1 representation: [0,0,0,0,1,0,0,0,0,1,0].
+            //    Paths IDs' 0-1 representation: [1,1,1,1,1,1,0,1,0].
+            //  The original feature vector [0.5888848,0.9360271,0.4721779] is transformed to three different tree-based feature vectors:
+            //    Trees' output values: [0.4545455,0.8].
+            //    Leave IDs' 0-1 representation: [0,0,0,1,0,0,0,0,0,0,1].
+            //    Paths IDs' 0-1 representation: [1,1,1,1,0,1,0,1,1].
+            //  The original feature vector [0.2737045,0.2919063,0.4673147] is transformed to three different tree-based feature vectors:
+            //    Trees' output values: [0.4545455,0.1111111].
+            //    Leave IDs' 0-1 representation: [0,0,0,1,0,0,1,0,0,0,0].
+            //    Paths IDs' 0-1 representation: [1,1,1,1,0,1,0,1,1].
+        }
+
+        private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count, int seed=0)
+        {
+            var random = new Random(seed);
+            float randomFloat() => (float)random.NextDouble();
+            for (int i = 0; i < count; i++)
+            {
+                var label = randomFloat() > 0.5f;
+                yield return new DataPoint
+                {
+                    Label = label,
+                    // Create random features that are correlated with the label.
+                    // For data points with false label, the feature values are slightly increased by adding a constant.
+                    Features = Enumerable.Repeat(label, 3).Select(x => x ? randomFloat() : randomFloat() + 0.03f).ToArray()
+                };
+            }
+        }
+
+        // Example with label and 3 feature values. A data set is a collection of such examples.
+        private class DataPoint
+        {
+            public bool Label { get; set; }
+            [VectorType(3)]
+            public float[] Features { get; set; }
+        }
+
+        // Class used to capture the output of tree-base featurization.
+        private class TransformedDataPoint : DataPoint
+        {
+            // The i-th value is the output value of the i-th decision tree.
+            public float[] Trees { get; set; }
+            // The 0-1 encoding of leaves the input feature vector falls into.
+            public float[] Leaves { get; set; }
+            // The 0-1 encoding of paths the input feature vector reaches the leaves.
+            public float[] Paths { get; set; }
```

Comment:
Maybe:
// The 0-1 encoding of paths the input feature vector follows to reach the leaves.